lms/templates/calculator/toggle_calculator.html

```diff
@@ -22,9 +22,9 @@
           <ul id="calculator_input_help" class="help" aria-hidden="true">
             <li class="hint-item" id="hint-moreinfo">
                 <p><span class="bold">${_("For detailed information, see {math_link_start}Entering Mathematical and Scientific Expressions{math_link_end} in the {guide_link_start}edX Guide for Students{guide_link_end}.").format(
-                  math_link_start='<a href="http://edx-guide-for-students.readthedocs.org/en/latest/SFD_mathformatting.html">',
+                  math_link_start='<a href="http://edx.readthedocs.io/projects/edx-guide-for-students/en/latest/completing_assignments/SFD_mathformatting.html#math-formatting">',
```

Comment:
Why is one of these URLs at {{edx-guide-for-students.readthedocs.org}} but the other at {{edx.readthedocs.io/projects/edx-guide-for-students}}? I would have thought that the URL on this line would be:

    http://edx-guide-for-students.readthedocs.io/en/latest/completing_assignments/SFD_mathformatting.html#math-formatting